templates/api-blueprint.swig

```diff
@@ -102,6 +106,10 @@ FORMAT: 1A
 ### {{ resource.title|default('Resource') }} [{{ resource.href }}]
 
 {{ resource.description }}
```

Comment:
It is invalid according to the current spec since Copy elements are going to used instead of description for the specified elements.

Yes, swagger adapter needs to be changed.